app/services/moab_storage_root_reporter.rb

```diff
@@ -59,10 +64,10 @@ def moab_detail_csv_list(errors_only: false)
   # @return [String] the name of the CSV file to which the list was written
   # @raise [ArgumentError] if neither report_type nor filename is provided
   # @raise [RuntimeError] if the file to be written to already exists
-  def write_to_csv(lines, report_type: nil, filename: nil)
+  def write_to_csv(lines, report_type: nil, report_tag: nil, filename: nil)
```

Comment:
@jmartin-sul Can you add YARD documentation for the new param? (I know this has already been merged...)

Reply:
d'oh, sorry, i am just not in the habit of doing YARD doc, see https://github.com/sul-dlss/preservation_catalog/pull/1374